docs/platforms/python/integrations/celery/index.mdx

```diff
@@ -121,9 +129,11 @@ app = Celery("tasks", broker="...")
 def init_sentry(**_kwargs):
     sentry_sdk.init(...)  # same as above
 
+# ___PRODUCT_OPTION_START___ performance
 @app.task
 def debug_sentry():
     1/0
+# ___PRODUCT_OPTION_END___ profiling
```

Comment:
I don't think these should be here. Not sure why this snippet had onboarding options before.

```suggestion
@app.task
def debug_sentry():
    1/0
```